### PR TITLE
Move some browser docs to recommended practices

### PR DESCRIPTION
**Pull request by inancgumus**
The main reason for this change is to collect recommended practices for using the k6 browser under a single menu. + Putting the migration guide next to the other items in the menu overcrowds, make it hard to follow, and doesn't represent a cohesive structure.

We have usage examples already in the other API documents. These examples are more like "recommended practices". Also, "Recommended practices and examples" didn't fit the menu and became weird, like "Recommended practices an..."

### CODE REVIEW

**Comment by mdcruz** [comment]
We need to add some redirects so the old URL will not throw a 404 error:

- /using-k6-browser/selecting-elements/ -> /using-k6-browser/recommended-practices/selecting-elements/
- /using-k6-browser/examples/page-object-model/ -> /using-k6-browser/recommended-practices/page-object-model/

This needs to be updated on the `gatsby-node.js` file.

```
createRedirect({
    fromPath: '/using-k6-browser/selecting-elements/,
    toPath:
      '/using-k6-browser/recommended-practices/selecting-elements/',
    isPermanent: true,
  });
```

**Comment by inancgumus** [comment]
Thx, @mdcruz 👍 That makes sense!

See the change here: 394d1f79

**Comment by inancgumus** [comment]
Thx, @mdcruz 👍 That makes sense!

See the changes here: 394d1f79 and 03f5c7db

**Review by mdcruz — APPROVED** [review]
LGTM! 🚀
